platform/darwin/src/MGLMultiPoint.h

```diff
@@ -35,6 +35,22 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)getCoordinates:(CLLocationCoordinate2D *)coords range:(NSRange)range;
 
+/**
+ Updates one or more coordinates for the shape, which will instantaneously 
+ cause the shape to be redrawn if it is currently visible on the map.
+ 
+ @param coords The array of coordinates defining the shape. The data in this
+ array is copied to the object.
+ @param range The range of points to update. The `location` field indicates the
+ first point you are replacing, with `0` being the first point, `1` being
+ the second point, and so on. The `length` field indicates the number of
+ points to update. You can append to an existing array of coordinates
+ by specifying a range with a `location` at the end of the existing array 
+ and/or a `length` which extends past the end of the existing array. The array 
+ in _`coords`_ must be equal in number to the length of the range.
+ */
+- (void)setCoordinates:(CLLocationCoordinate2D *)coords range:(NSRange)range;
```

Comment:
Something didn’t feel right to me, and now I know why: we’re only allowing the developer to overwrite or extend the multipoint but not shrink it. They can splice _in_ but not splice _out_. We’re assuming that `coords`’ range is equal to `range.length`, whereas the length would have to be passed in explicitly.

I don’t think we should take the time right now to implement all the possible splicing operations, but we should try to be consistent with [naming convention for KVO indexed accessors](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/KeyValueCoding/Articles/AccessorConventions.html#//apple_ref/doc/uid/20002174-SW11), which NSMutableArray and others adhere to. Ideally, we’d call this method `-replaceCoordinatesInRange:withCoordinates:count:` and have separate `-setCoordinates:count`, `-insertCoordinates:count:atIndex:`, and `-removeCoordinates:range:` methods. But for now, let’s call this method `-replaceCoordinatesInRange:withCoordinates:`. When we eventually implement `-replaceCoordinatesInRange:withCoordinates:count:`, we can deprecate this method.